docs/test_plan.md

```diff
@@ -0,0 +1,59 @@
+|3|Click "Catalog" in the header to return to the home page|
+|4|Select `Author (keyword)` from the dropdown in the Search bar|
+|5|Enter `john d clark` in the search box and press the enter key or click the search button|You are taken to a search results page with a number of matching results|
+
+## Record Page
+
+|Step|Action|Expected result|
+|---|---|---|
+|1|Go to the Record page for [an item](https://catalog-staging.princeton.edu/catalog/99111163333506421)|The application displays a page with details on the book It's all Greek|
+|2|There should be a section with author, format, language, etc. for the item|
+|3|There should be a section with the holding location, call number, availability information, and a request button|
+|4|Click the `Where to find it` button|A modal should pop up with a map of firestone with a marker indicating the shelving location of the item|
+|5|There should be a section with detailed information, including subjects, a summary, and a link to the staff view|
+|5|Click the `Staff view` link|The detailed MARC record should be displayed|
+
+## Request Form
+
+|Step|Action|Expected result|
+|---|---|---|
+|1|Without an active session, go to the Record page for [an item](https://catalog-staging.princeton.edu/catalog/99111163333506421)|The application displays a page with details on the book It's all Greek|
+|2|Click on the `Request` button|You should be redirected to the login page|
+|3|Click on the `Log in with netID` button and follow the CAS prompts|The request form should display with the item selected and the delivery options unselected. The `Request this Item` button should be disabled|
+|4|Click the `Physical Item Delivery` radio element|The Pick-up location should display Firestone Library and the `Request this Item` button should be enabled|
+|5|Click the checkbox in the `Select` column|The `Request this Item` button should be disabled|
+|6|Click the checkbox in the `Select` column to re-enable the `Request this Item` button|
+|7|Click the `Request this Item` button to submit the request|A success notification with the text `Item has been requested for pick-up` should appear|
+
+## Advanced Search
+
+|Step|Action|Expected result|
+|---|---|---|
+|1|From the [home page](https://catalog-staging.princeton.edu/) click on the advanced search link next to the basic search box|The application should navigate to the advanced search form|
+|2|In the first text input enter `cats` and click the `Search` button|Search results with approximately 45,000 items should display|
+|3|Click the `Edit search` button|You should return to the advanced search form|
+|4|In the second text input next to `Author/Creator` enter `warhol` and click the `Search` button|Search results with about 9 items should display|
+|5|Click the `Edit search` button|You should return to the advanced search form|
+|6|Select `French` from the `Language` dropdown and click the `Search` button|Search results with 2 items should display|
```

Comment:
In step 5 can we test as a result not the number of the results but the selected filters on the screen? The results will not be the same because we update the index. 
In this case the filter is:
![filter author/creator=Warhol and language=French](https://github.com/user-attachments/assets/3ad00e70-cdf9-4409-a0a2-5faeca924649)

Reply:
Done!